src/main/resources/assets/js/components/AppHealthComponent.jsx

```diff
@@ -0,0 +1,78 @@
+/** @jsx React.DOM */
+
+var React = require("react/addons");
+
+var AppHealthComponent = React.createClass({
+  displayName: "AppHealthComponent",
+
+  getHealthData: function () {
+    var model = this.props.model;
+
+    var tasksWithUnknownHealth = Math.max(
+      model.get("tasksRunning") -
+      model.get("tasksHealthy") -
+      model.get("tasksUnhealthy"),
+      0
+    );
+
+    var healthData = [
+      {quantity: model.get("tasksHealthy"), className: "health-bar-healthy"},
+      {quantity: model.get("tasksUnhealthy"), className: "health-bar-unhealthy"}, /* jscs:disable maximumLineLength */
```

Comment:
Of course I don't. But here the line is 81 characters and breaking the line introduces visual clutter much much worse than keeping this 81 char line.